.github/workflows/integtest.yaml

```diff
@@ -97,6 +97,12 @@ jobs:
           docker ps -a
           redis-cli -h 127.0.0.1 -p 8000 ping
           redis-cli -h 127.0.0.1 -p 8000 cluster nodes
+      - name: local vars
+        working-directory: ${{ matrix.app_to_test }}
+        run: |
+          if [[ -f test/integ_test_hooks/local-vars.sh ]]; then
+            test/integ_test_hooks/local-vars.sh
```

Comment:
nit: it's a bit inconsistent if this script is executable, but the others in this dir aren't and need to be explicitly invoked with `bash <path>`.

Do you have a preference? If you like this style (I personally do), I can put in a small followup PR to convert the others.

Reply:
since its local i also think we need to have a way of running it locally and its more intuitive to just have the command be executable in my opinion